src/to_dom.js

```diff
@@ -116,17 +117,31 @@ export class DOMSerializer {
   // Render an [output spec](#model.DOMOutputSpec) to a DOM node. If
   // the spec has a hole (zero) in it, `contentDOM` will point at the
   // node with the hole.
-  static renderSpec(doc, structure) {
+  static renderSpec(doc, structure, parentXmlns = null) {
     if (typeof structure == "string")
       return {dom: doc.createTextNode(structure)}
     if (structure.nodeType != null)
       return {dom: structure}
-    let dom = doc.createElement(structure[0]), contentDOM = null
+    let contentDOM = null
+    let [xmlns, tag] = structure[0].split(' ', 2)
```

Comment:
Could you do something involving `structure[0].indexOf(" ")` to check for the presence of a space, to avoid allocating another array for every element?